src/pages/derivx/_what-is-derivx.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Localize } from 'components/localization'
+import { SectionContainer, Container, Flex } from 'components/containers'
+import { Header, Text } from 'components/elements/typography'
+import device from 'themes/device'
+
+const StyledSectionContainer = styled(SectionContainer)`
+    border-bottom: solid 1px var(--color-grey-2);
+
+    @media ${device.tablet}{
+        border-bottom: unset;
+    }
+`
+
+const WhatIsDeriv = () => {
+    return (
+        <StyledSectionContainer>
+            <Container>
+                <Flex width="792px" fd="column" ai="center" jc="center">
+                    <Header type="page-title" align="center">
+                        <Localize translate_text="What is Deriv X" />
```

Comment:
Is there a question mark here ?

Reply:
According to design there's no question mark here